decidim-admin/app/controllers/decidim/admin/component_permissions_controller.rb

```diff
@@ -55,12 +75,19 @@ def other_authorizations_for(action)
         )
       end
 
+      def resource
+        @resource ||= if params[:resource_id] && params[:resource_name]
+                        manifest = Decidim.find_resource_manifest(params[:resource_name])
+                        manifest&.resource_scope(component)&.find_by(id: params[:resource_id])
```

Comment:
Coverage reports these lines as not tested, is that right?

Reply:
Yes, I'm completing participatory spaces system tests to cover this case :+1:

Reply:
@leio10 nice, tell me when it's ready for a final review! 😄